### PR TITLE
FIX: COCOA: controls position error in CopyMoveDlg

### LINKED ISSUE #340: Some options are unreachable on the copy / move (F5/F6)
On Mac using v1.1.0 alpha build 10045 2022/01/06

When one click on the options button you cant get to the options in the bottom, the window is not resizable nor can scroll up/down the options there.

image

image

### PR DESCRIPTION
**Pull request by rich2014**
fix #340 and #341 
tested on MacOs and Windows

### CODE REVIEW

**Comment by rich2014** [comment]
image

File: src/fcopymovedlg.lfm [hunk 1]
```diff
@@ -10,6 +10,7 @@ object frmCopyDlg: TfrmCopyDlg
   ActiveControl = edtDst
   AutoSize = True
   BorderIcons = [biSystemMenu, biMaximize]
+  BorderStyle = bsSingle
```
Comment (alexx2000) [inline src/fcopymovedlg.lfm:13]: Is it a mandatory change? Can it be done only for Cocoa (via code)?

Comment (rich2014) [reply src/fcopymovedlg.lfm:13]: 1. it's not a mandatory requirement, not even for cocoa
2. it's just a convention, the dialog is fixed size like other dialogs in DC
3. CopyMoveDlg was originally set to be resizable, is there any special consideration?

Comment (alexx2000) [reply src/fcopymovedlg.lfm:13]: Many dialogs in DC are resizable. Only some dialogs has fixed size. User may want to resize CopyMoveDlg if full target path is very long.

Comment (rich2014) [reply src/fcopymovedlg.lfm]: THX.
form BorderStyle setting restored.
check it please.

**Pull request merged**